releases/release-1.34/README.md

```diff
@@ -0,0 +1,110 @@
+
+## Timeline
+
+| **What**                                                                               | **Who**                       | **When**                                                                                                        | **Week** | **Release Signal**                                                       |
+|----------------------------------------------------------------------------------------|-------------------------------|-----------------------------------------------------------------------------------------------------------------|----------|--------------------------------------------------------------------------|
+| Start of Release Cycle                                                                 | Lead                          | Monday 19th May 2025                                                                                            | week 1   | [master-blocking], [master-informing]                                    |
+| Start Enhancements Tracking                                                            | Enhancements Lead             | Monday 19th May 2025                                                                                            | week 1   |                                                                          |
+| Schedule finalized                                                                     | Lead                          | Friday 23rd May 2025                                                                                            | week 1   |                                                                          |
+| Team finalized                                                                         | Lead                          | Friday 23rd May 2025                                                                                            | week 1   |                                                                          |
+| Begin APAC-friendly meetings                                                           | Lead                          | Wednesday 28th May 2025                                                                                         | week 2   |                                                                          |
+| Start Release Notes Draft                                                              | Docs Lead                     | Wednesday 4th June 2025                                                                                         | week 3   |                                                                          |
+| 1.34.0-alpha.1 released                                                                | Branch Manager                | Wednesday 11th June 2025                                                                                        | week 4   |                                                                          |
```

Comment:
I have shifted all the release cuts from Tuesday to Wednesday for two reasons (which were brought up in the retro):

1. Tuesdays are for patch releases, so having the main cut after it makes sense and reduced workload (on people/tooling/infra)
2. Having the cut on Tuesday doesn't leave much for an issue fix if it was discovered Friday eod, this gives somewhat more flexibility. (However, it was mentioned that we can always delay the cut for this particular point).

Reply:
I strongly agree on the first point, i.e. decoupling the patch releases from the prereleases, these shouldn't be scheduled for the same day. Especially given that some part of the release process are not concurrent (the image promotion part).

I initially thought about doing the prereleases on Tuesday, and the patch releases on Wednesday, but that would be some more work to get it adopted. So let's try with the preleases on Wednsday, and the patch releases on Tuesday (as @Vyom-Yadav proposed here).